### ISSUE TITLE
async howto implies Deferreds solve concurrency issues

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2718]
|[<img alt="itamarst's avatar" src="https://avatars.githubusercontent.com/u/3266662?s=50" width="50" height="50">](https://github.com/itamarst)| @itamarst reported|
|-|-|
|Trac ID|trac#2718|
|Type|defect|
|Created|2007-06-25 11:55:57Z|

```
It is the second class of concurrency problem — non-computationally intensive tasks that involve an appreciable delay
— that Deferreds are designed to help solve. Functions that wait on hard drive access, database access, and network
access all fall into this class, although the time delay varies.
```

...

```
The basic idea behind Deferreds, and other solutions to this problem, is to keep the CPU as active as possible.
```

I think this howto is somewhat misleading and sorta implies that Deferreds have something to do with Twisted's concurrency model, a confusion many new users have.

<details><summary>Searchable metadata</summary>

```
trac-id__2718 2718
type__defect defect
reporter__itamarst itamarst
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__documentation documentation
time__1182772557000000 1182772557000000
changetime__1183040841000000 1183040841000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_1"></a>|@therve set status to `closed`|
|-|-|

Duplicate of [#2719](https://github.com/twisted/twisted/issues/2719).

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
